docs/send-data/opentelemetry-collector/remote-management/processing-rules/mask-rules-windows.md

```diff
@@ -1,36 +1,40 @@
 ---
 id: mask-rules-windows
-title: Mask Rules for Windows Source Template (Beta)
-sidebar_label: Mask Rules for Windows
+title: Mask Rules for the Windows Source Template
+sidebar_label: Mask Rules - Windows Source Template
 description: Create a mask rule to replace an expression with a mask string.
 ---
-<head>
-  <meta name="robots" content="noindex" />
-</head>
-
-<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
```

Comment:
Although you removed the beta tagging, you still need to add this article to the sidebar and the index file.

Reply:
Wow, great catch!